samples/build.pl

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/perl
```

Comment:
can we avoid this wrapper altogether, please? Just default configuration of cmake should be enough.